backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/utils/ApplicationHandler.kt

```diff
@@ -166,8 +166,8 @@ class ApplicationHandler(
             require(organizations.isNotEmpty()) {
                 "Work at organizations list cannot be empty if application is already verified"
             }
-            require(organizations.all { it.organization.category.shortText == "Sport" }) {
-                "All organizations must be of category Sport if application is already verified"
+            require(organizations.all { it.organization.category.shortText == "sports" }) {
```

Comment:
@connium if we store values instead of labels for dropdowns we would have also to inform verein 360 that the value they have to send us will slightly change with the next release from `Sport` to `sports`

Reply:
I don't think this would raise much of effort for them, as I'm not quite sure whether they even started with the implementation.
Is this this only change, that affects them?

I think, in the worst case, we could even add a mapping, as this affects exactly a single interface on our side.
Do you know the developer for the verein360 stuff? If so, can you please contact her/him? Otherwise I'd have to write this to our contact person at the BLSV.

Reply:
no unfortunately i don't have contact to a dev.
I think it will be the only change
I could also make a mapping for the data they send to us from `Sport` to `sports` but i think such a small change shouldn't be a problem for them to adjust

Reply:
I can see a small issue if they send as a label and not a value:
Technically they can send requests with isAlreadyVerified: false
In this case, checking it.organization.category.shortText == “sports” will not be performed, and if they submit something else, it will show up in the UI as an unresolved translation.

Reply:
> I think, in the worst case, we could even add a mapping

Actually I don't see why we need to change the backend here?
We could make translatable labels in the UI form. But when the request is sent directly to the API, why not leave it as it is? It just won't be translated, but it will display the value from the database.

Reply:
Additional thought:
If we want to somehow apply localization to data from direct API requests, I think that should be a different task and concept if necessary.
Because we also have addresses (Deutschland vs Germany) and other fields. Why would we only translate category?

Reply:
I think dropdowns are the most important things since they are selections, also radio button selections.
Other values are directly typed values and can be stores as they were typed.
But maybe we also have to handle this in the future

Reply:
> Maybe i just don't get the point

1. User fills the form via /beantragen and selects category, for example, "Sport". We use localization to display the category name in the right language to the user.
2. We send the translated value in the request to the backend --> the “Sport” value is stored in the database
3. In the application overview we don't translate the category name, but use the one from the database.
For Verein360 requests it will be the value from the request.
For normal application flow, this will be the value translated according to the localization of the user who created the application.

Reply:
So my idea is to display the category in the application overview in the language in which it was submitted.
Because we basically do that with other fields as well - country, city names, other free input fields.

Reply:
So therefore we would have to check the translations json also in the backend.
Okay i'm okay that we just send the label for now and think about a general solution @seluianova 
Since we don't use another language right now its not super important to tackle this now

Reply:
alright! good to go then :)